main.go

```diff
@@ -50,6 +50,7 @@ var (
 	listenAddress = kingpin.Flag("web.listen-address", "The address to listen on for HTTP requests.").Default(":9115").String()
 	timeoutOffset = kingpin.Flag("timeout-offset", "Offset to subtract from timeout in seconds.").Default("0.5").Float64()
 	configCheck   = kingpin.Flag("config.check", "If true validate the config file and then exit.").Default().Bool()
+  historyLimit  = kingpin.Flag("historyLimit", "The maximum amount of items to keep in the history.").Default("100").Int()
```

Comment:
Why not uint? This also needs a gofmt.

Reply:
I went with int because I thought it looked cleaner to avoid the having a cast on history.go:54 and negative values will be set to 100 by history.go:40. I have no problem changing it though.

Formatting issue was because I tried to use the github merge ui. Will fix with the uint commit.